wordpress-mobile/WordPress-iOS
# Issue/7664 media settings design

Fixes #7644 and #7707. There weren't a lot of issues here, just changing the slider for video resolution to be a tableview.

To test:
1. Change video resolution
2. Verify that changing the resolution saves the `MediaSettings().maxVideoSettings` value

![simulator screen shot aug 21 2017 7 57 00 pm](https://user-images.githubusercontent.com/1158819/29517570-b2d94f5e-86b0-11e7-8817-cd3f8ac10dc5.png)

![simulator screen shot aug 21 2017 7 56 57 pm](https://user-images.githubusercontent.com/1158819/29517569-b2cda5be-86b0-11e7-9d8d-1876b5acfa2e.png)
